Fixing content bug in step 3 notice box (#2757)
elastic/observability-docs: docs/en/observability/ingest-traces.asciidoc

== Step 3: Install and run an {agent} on your machine

****

> As {ecloud} spins up an {agent} instance automatically or 
> self-managed users installed an {agent} instance manually in Step 1, 
> this step is optional only for users who need to add a new Elastic Agents.
> If you only need to use APM features, {ess} and self-managed users both should skip this step.

Reviewer: ```suggestion
This step is optional for both {ess} and self-managed users as
{ecloud} spins up an {agent} instance automatically, and self managed users installed an {agent} instance manually in Step 1.
Unless you need to add additional {agent}s, skip this step.
```
